docs/Test-Path2.md

```diff
@@ -0,0 +1,81 @@
+---
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Test-Path2
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```

Comment:
Determines if a ll elements of a path exist.  The path is not limited to 256 characters.